README.md

```diff
@@ -121,6 +123,11 @@ This is what the default configuration looks like:
   mongodbUrl: 'mongodb://localhost:27017/poet',
   ipfsUrl: 'http://localhost:5001',
   insightUrl: 'https://test-insight.bitpay.com/api',
+  bitcoinUrl: '',
```

Comment:
What is this bitcoinUrl, what is it and where do I get it? Maybe explain those things a bit more in the readme.

Same for the other configuration settings.

Reply:
Don't we need to remove `insightUrl` now too?

Reply:
@geoffturk `insightUrl` is still used by the BlockchainWriter, this is just the Reader conversion.

Reply:
@kennylavender now that you mention it, you made me remember something.

This experience is rather painful and not in line with the other dependencies.

I should add a bitcoin-core running in testnet to the docker-compose file and change the default configuration so it works out-of-the-box with that docker image of bitcoind, as we're doing with IPFS, RabbitMQ and MongoDB.

I think I'll start working on that and then update the README to reflect so.